<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProductShell</class>
 <widget class="QWidget" name="ProductShell">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>298</width>
    <height>509</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelName">
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>1</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelSummary">
     <property name="text">
      <string>Общий результат:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelCyclesPassedLeft">
     <property name="text">
      <string>Циклов прошло/осталось:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelTimePassedLeft">
     <property name="text">
      <string>Времени прошло/осталось:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="textEditShell"/>
   </item>
   <item>
    <widget class="QPushButton" name="buttonPause_2">
     <property name="text">
      <string>Очистить консоль</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="buttonTest">
     <property name="text">
      <string>Тест</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="buttonReset">
     <property name="text">
      <string>Сброс</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="buttonPause_3">
     <property name="text">
      <string>Пауза</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
